agenta-cli/agenta/sdk/agenta_init.py

```diff
@@ -179,12 +187,12 @@ def pull(self, config_name: str = "default", environment_name: str = None):
         if self.persist:
             try:
                 if environment_name:
-                    config = client.configs.get_config(
+                    config = self.client.configs.get_config(
                         base_id=self.base_id, environment_name=environment_name
                     )
 
                 else:
-                    config = client.configs.get_config(
+                    config = self.client.configs.get_config(
                         base_id=self.base_id,
                         config_name=config_name,
                     )
```

Comment:
def dump(self):
"""
Returns all the information about the current version in the configuration. 
{environment:"",
variant_id:""
variant_revision:"",
configuration: self.all()}